en/03_Drawing_a_triangle/04_Swap_chain_recreation.md

```diff
@@ -18,23 +18,15 @@ void recreateSwapChain() {
 
     createSwapChain();
     createImageViews();
-    createRenderPass();
-    createGraphicsPipeline();
     createFramebuffers();
 }
 ```
 
 We first call `vkDeviceWaitIdle`, because just like in the last chapter, we
 shouldn't touch resources that may still be in use. Obviously, the first thing
 we'll have to do is recreate the swap chain itself. The image views need to be
-recreated because they are based directly on the swap chain images. The render
-pass needs to be recreated because it depends on the format of the swap chain
-images. It is rare for the swap chain image format to change during an operation
-like a window resize, but it should still be handled. Viewport and scissor
-rectangle size is specified during graphics pipeline creation, so the pipeline
-also needs to be rebuilt. It is possible to avoid this by using dynamic state
-for the viewports and scissor rectangles. Finally, the framebuffers directly
-depend on the swap chain images.
+recreated because they are based directly on the swap chain images. Finally, the 
+framebuffers directly depend on the swap chain images.
```

Comment:
```suggestion
recreated because they are based directly on the swap chain images. Finally, the 
framebuffers directly depend on the swap chain images, and thus must be recreated as well.
```